modules/apps/commerce/commerce-frontend-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"dependencies": {
-		"@clayui/alert": "3.26.0",
+		"@clayui/alert": "3.5.1",
```

Comment:
Is this intentional? Any back-story to it? (As you already saw, SF won't let you do this.)

Reply:
Hello Greg i'm very glad about this review. I started this branch from master and i overrode the commerce-frontend-js folder. This means I found a more recent version of this dependency on my branch then on master branch

Reply:
Hey @wincent I don't understand what you mean, do you mean that this version change should not be done and we should use the 3.26.0 version? Thanks!

Reply:
@andrea-ale-sbarra: You don't have a choice — it has to be 3.26.0 because:

```
format-source-files:
     [java] java.lang.Exception: Found 3 formatting issues:
     [java] 3: ./modules/apps/commerce/commerce-frontend-js/package.json expected:<..."@clayui/alert": "3.[26.0]",
     [java] 		"@clayui/autoco...> but was:<..."@clayui/alert": "3.[5.1]",
     [java] 		"@clayui/autoco...>
```

(And the reason we have that rule is because we don't want multiple redundant versions of Clay packages floating around the repo.)